docs/samples/Microsoft.ML.Samples.FSharp/Dynamic/Transforms/Text/ApplyCustomWordEmbedding.fs

```diff
@@ -0,0 +1,79 @@
+﻿module Microsoft.ML.Samples.Dynamic.ApplyCustomWordEmbeddings
+open Microsoft.ML
+open Microsoft.ML.Data
+open System
+open System.IO
+
+type TextData(value:string) = 
+    member val Text = value with get,set
+    new() = TextData("")
+
+type TransformedTextData(value: string) = 
+    inherit TextData(value)
+    member val Features:float32 array = Array.empty with get, set
+
+    new() = TransformedTextData("")
+
+let Example = 
```

Comment:
I'd consider moving all of this file to Program.fs. However if the current project structure is maintained I would do 
```
let example() =
    ...
```
Given it outputs to the console and how it's referenced in the `main` function.